api_contracts/contracts/award/AwardProfile.md

```diff
@@ -0,0 +1,115 @@
+
+### SubAwards [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + award_id: 123 (optional, string)
+            The internal id of the award to filter on. If not included, all sub-awards are returned.
+        + limit: 15 (optional, number)
+            The number of results to include per page. Defaults to 10.
+        + page: 1 (optional, number)
+            The page of results to return based on the limit. Defaults to 1.
+        + sort: subaward_number (optional, string)
+            The field results are sorted by. Defaults to `subaward_number`. One of `subaward_number`, `description`, `action_date`, `amount`, `recipient_name`.
+        + order: desc (optional, string)
+            The direction results are sorted by. Defaults to descending.
+        
++ Response 200 (application/json)
+    + Attributes
+        + results (array[SubAwardResult], fixed-type)
+        + page_metadata (PageMetaDataObject)
+        
+## Transactions [/api/v2/transactions/]
+
+This endpoint returns a list of transactions, their amount, type, action date, action type, modification number, and description.
+
+Results with award type codes corresponding to loans will follow the `LoanTransactionResult` format. All others will follow the `TransactionResult` format.
+
+### Transactions [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + award_id: 123 (optional, string)
+            The internal id of the award to filter on. If not included, all transactions are returned.
+        + limit: 15 (optional, number)
+            The number of results to include per page. Defaults to 10.
+        + page: 1 (optional, number)
+            The page of results to return based on the limit. Defaults to 1.
+        + sort: action_date (optional, string)
+            The field results are sorted by. Defaults to `action_date`. One of `modification_number`, `action_date`, `federal_action_obligation`, 
+            `face_value_loan_guarantee`, `original_loan_subsidy_cost`, `action_type_description`, or `description`.
+        + order: desc (optional, string)
+            The direction results are sorted by. Defaults to descending.
+        
++ Response 200 (application/json)
+    + Attributes
+        + results (array[TransactionResult], fixed-type)
+        + page_metadata (PageMetaDataObject)
+
+        
+# Data Structures
+
+## SubAwardResult (object)
++ id: `1` (required, string)
+    The internal sub-award id.
++ subaward_number: 2-A (required, string)
+    The sub-award id.
++ description: description (required, string)
++ action_date: 1999-01-15 (required, string) 
+    Action date in the format `YYYY-MM-DD`.
++ amount: 1234.56 (required, number)
+    Monetary value of the sub-award.
++ recipient_name: Recipient A (required, string)
+    
+## TransactionResult (object)
++ id: `1` (required, string)
+    The internal transaction id.
++ type: A (required, string)
+    Award type code
++ type_description BPA (required, string)
```

Comment:
Missing a colon between `type_description` and `BPA` so the tests fail. See last line in snippet below

```
...
expected: 
headers: 
    Content-Type: application/json

body: 
{
  "results": [
    {
      "id": "1",
      "type": "A",
      "type_description BPA": "",
...
```